libnmstate/nm/plugin.py

```diff
@@ -17,6 +17,9 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 from operator import itemgetter
+from warnings import warn
+from distutils.version import StrictVersion
+from gi.repository import NM
```

Comment:
Please, sort the imports by alphabetical order.

Reply:
Done, thank you!